python/test/t_MetaModelValidation_std.py

```diff
@@ -104,8 +104,9 @@
 metaModel = ot.SymbolicFunction(["x"], ["x - x^3/6.0 + x^5/120.0", "cos(1.2*x)"])
 x = dist.getSample(1000)
 y = model(x)
-val = ot.MetaModelValidation(x, y, metaModel)
-q2 = val.computePredictivityFactor()
+metamodelPredictions = metaModel(x)
+val = ot.MetaModelValidation(y, metamodelPredictions)
+q2 = val.computeR2Score()
```

Comment:
Should it be r2Score (even if it is in the Python test) ?

Reply:
Ok. Fixed at 05c09aec6.